docs/pages/eas-update/rollouts.mdx

```diff
@@ -2,24 +2,29 @@
 title: Rollouts
 hideTOC: true
 ---
+
 import { Terminal } from '~/ui/components/Snippet';
 
+> This feature is in Developer Preview. It's not advised to use this feature in any production environments.
+
+Rollouts incrementally deploy updates on a new branch to a specific channel. With rollouts, we can roll out updates from one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to our production environment.
 
-Rollouts allow deploying a new branch to a specific channel incrementally. With rollouts, we can roll out one branch to a percentage of end users and leave the remaining percentage of users on the current branch. This is useful when testing a new feature to minimize the risk of introducing bugs or other issues to your production environment.
+To start a rollout, we use `npx eas-cli@latest` or ensure we have version \<TODO\> or above for `eas-cli`. Then we run the following command:
```

Comment:
```suggestion
## Start a rollout

To start a rollout, use `npx eas-cli@latest` or ensure you have version \<TODO\> or above for `eas-cli`. Then, run the following command:
```

- Let's add a section title here
- Let's also not forget about adding the `TODO` version before publishing this doc 😄 
- Also, as per out writing style guide, [we use second person voice and keep "we" reserved for us, that is, the Expo team](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person). I've made these edits to follow this pattern in the rest of the document as well for consistency reasons.